frontend/packages/ux-editor/src/components/config/componentSpecificContent/Button/ButtonComponent.test.tsx

```diff
@@ -35,8 +34,8 @@ describe('ButtonComponent', () => {
       type: ComponentType.NavigationButtons,
       showBackButton: true,
       textResourceBindings: {
-        next: 'next',
-        back: 'back',
```

Comment:
With this changing the default layout of buttons will be without text , just a blue color, I am not sure if this ok

Reply:
Done 👍